src/common/ceph_time.h

```diff
@@ -499,6 +500,10 @@ namespace std {
 	std::chrono::duration_cast<timespan>(z) :
 	timespan(-z.count());
     }
+    inline timespan totimespan(signedspan z) {
```

Comment:
I'd rather this be named to_timespan (to match all the other to_* functions we use all over the place) or make_timespan (to match the various make_timespan overloads.)

Reply:
Sure, I'll change that.